### PR TITLE
Fix paths and add copyright boilerplate

### PR DESCRIPTION
**Pull request by jcrossley3**
Minor fixes resulting from the move

### CODE REVIEW

**Comment by jchesterpivotal** [comment]
I would make a joke here about /symboliclgtm, but then I realised that's why github added emoji.

File: test/e2e/knative_serving_test.go [hunk 1]
```diff
@@ -4,8 +4,8 @@ import (
 	"testing"
```
Comment (houshengbo) [inline test/e2e/knative_serving_test.go]: We do not have header in this file.

Comment (jcrossley3) [reply test/e2e/knative_serving_test.go]: Good catch!

**Pull request merged**